src/components/AddPlaidBankAccount.tsx

```diff
@@ -62,6 +62,9 @@ type AddPlaidBankAccountProps = AddPlaidBankAccountOnyxProps & {
     /** Is displayed in new VBBA */
     isDisplayedInNewVBBA?: boolean;
 
+    /** Is displayed in new enable wallet flow */
+    isNewWalletFlow?: boolean;
```

Comment:
```suggestion
    isDisplayedInWalletFlow?: boolean;
```
How about making the name similar to `isDisplayedInNewVBBA`